HULKs/hulk
# Small improvements in line detection

## Why? What?

- Allow to disable segment edge type checks
- Calculate sobel gradient over pixels with larger stride (increase interesting region and hopefully noise)
- Render edge types in line detection overlay

![image](https://github.com/user-attachments/assets/dd3b6598-ac91-49c3-8d2c-b0032bb6b060)

## ToDo / Known Issues

None

## Ideas for Next Iterations (Not This PR)

None

## How to Test

Deploy, open line detection panel, see that everything is great. Also you could change the stride parameter to see the difference.
